element/index.js

```diff
@@ -114,3 +114,26 @@ export function switchChildrenNodeName( children, nodeName ) {
 		return createElement( nodeName, { key: index, ...props }, childrenProp );
 	} );
 }
+
+/**
+ * Composes multiple higher-order components into a single higher-order component. Performs right-to-left function
+ * composition, where each successive invocation is supplied the return value of the previous.
+ *
+ * @param {...Function} hocs The HOC functions to invoke.
+ * @return {Function}        Returns the new composite function.
+ */
+export { flowRight as compose };
+
+/**
+ * Returns a wrapped version of a React component's display name.
+ * Higher-order components use wrapDisplayName().
+ *
+ * @param {Function|Component} BaseComponent used to detect the existing display name.
+ * @param {String} wrapperName Wrapper name to prepend to the display name.
+ * @return {String}            Wrapped display name.
+ */
+export function wrapperDisplayName( BaseComponent, wrapperName ) {
```

Comment:
Minor: Something about the function name I'm not a fan of, doesn't really indicate what it's for; wondering if it's useful enough a utility to simply retrieve a usable component name i.e. `getDisplayName` and concatenate manually when we need to; otherwise maybe `getWrapperDisplayName` ?

Reply:
Let's call it `getWrapperDisplayName` and revisit later if necessary.